website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,189 @@
+## Usage
+
+### When to use
+
+- When the list of options is large and search is required to help filter the list.
+- When list items are displayed in groups.
+- When complex, structured content is required to be displayed within the options.
+
+### When not to use
+
+- To display a list of links (or buttons) in a menu, consider the [HDS Dropdown](/components/dropdown) component instead.
+- When a simple select is needed for a form, consider the [HDS Select](/components/form/select) form component.
+- As a means to filter an array of objects within a data set; e.g., when presented in a tabular format or list. Instead, consider other [filtering patterns](/patterns/filter-patterns).
+
+## Components
+
+### Single
+
+Selection is limited to a single option item.
+
+![Single selection example with list open](/assets/components/form/super-select/super-select-single-variant-text.png =504x*)
+
+### Multiple
+
+Multiple options can be selected.
+
+![Multiple selection example with list open](/assets/components/form/super-select/super-select-multiple-variant.png =504x*)
+
+## Trigger value states
```

Comment:
We might want to discuss this as a team, but something that @didoo brought up in our work for the RichTooltip was avoiding the use of "trigger" because it may elicit a response from those with PTSD. We've opted to use toggle instead. I'm not sure of the data behind this, but it probably warrants a larger discussion. Non-blocker as there are probably other instances of this across the website.

Reply:
I believe we also use "toggle" for the dropdown. I'm all good with changing this for better empathetic language.